### PR TITLE
4.8.8 tiling (squares and octagons), angle increase (parquet style), save to json

### PR DESCRIPTION
**Pull request by radiodario**
Hey, here's the 4.8.8 tiling, and the option to vary the angles and deltas along the way. Also i've been using the code to plot some posters, so i've added a button that exports the lines to a json file.

screen shot 2017-02-05 at 03 52 02

thanks!

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -21,10 +25,22 @@
...
         <br />
         show grid: <input id="showGrid" type="checkbox" checked>
+        <br />
+        angle increase: <input id="angleIncrease" type="range" min="0" max="2.0" step="0.01" value="0">
+        delta increase: <input id="deltaIncrease" type="range" min="0" max="2.0" step="0.01" value="0">
+        change cycle: <input id="cycleIncrease" type="range" min="0" max="6.28" step="0.1" value="0">
     </div>
 </body>
-
+<script src="//localhost:35729/livereload.js"></script>
```
Comment (shiffman) [inline index.html]: Can you remove the live reload references?

**Review by shiffman — CHANGES_REQUESTED** [review]
This is so fantastic!  I saw your tweet too, so amazing, please continue to update me about your work. If you have a chance to remove the live reload stuff I'll merge!

**Comment by radiodario** [comment]
Hey sorry, this comes from merging @alvarobyrne 's changes, they are using livereload. 

Also i've got a branch with a version that only draws when a value from the GUI changes, is that something you'd like me to push? That way you can have way bigger canvases without slowdown

**Comment by shiffman** [comment]
@radiodario yes, I was planning to implement that myself so go right ahead!  You can make that a separate pull request.

**Review by shiffman — APPROVED** [review]

**Pull request merged**